docs/source/install.rst

```diff
@@ -0,0 +1,232 @@
+|                                           | invocations of           |
+|                                           | install.sh it is not     |
+|                                           | necessary to rebuild the |
+|                                           | dependencies.            |
++-------------------------------------------+--------------------------+
+| ``./install.sh -c``                       | Build library and client |
+|                                           | in your local directory. |
+|                                           | It is assumed the        |
+|                                           | dependencies have been   |
+|                                           | built.                   |
++-------------------------------------------+--------------------------+
+| ``./install.sh -idc``                     | Build library            |
+|                                           | dependencies, client     |
+|                                           | dependencies, library,   |
+|                                           | client, then build and   |
+|                                           | install the hipBLAS      |
+|                                           | package. You will be     |
+|                                           | prompted for sudo        |
+|                                           | access. It is expected   |
+|                                           | that if you want to      |
+|                                           | install for all users    |
+|                                           | you use the -i flag. If  |
+|                                           | you want to keep hipBLAS |
+|                                           | in your local directory, |
+|                                           | you do not need the -i   |
+|                                           | flag.                    |
++-------------------------------------------+--------------------------+
+| ``./install.sh -ic``                      | Build and install        |
+|                                           | hipBLAS package, and     |
+|                                           | build the client. You    |
+|                                           | will be prompted for     |
+|                                           | sudo access. This will   |
+|                                           | install for all users.   |
+|                                           | If you want to keep      |
+|                                           | hipBLAS in your local    |
+|                                           | directory, you do not    |
+|                                           | need the -i flag.        |
++-------------------------------------------+--------------------------+
+
+Dependencies
+============
+
+Dependencies are listed in the script install.sh. The -d flag to install.sh installs dependencies.
+
```

Comment:
Change "The -d flag to install.sh installs dependencies" to "Use the install.sh flag -d to install dependencies"